en/gps_compass/rtk_gps_cuav_c-rtk2.md

```diff
@@ -0,0 +1,115 @@
+# CUAV C-RTK2
+
+The [CUAV C-RTK2 receiver](http://doc.cuav.net/gps/c-rtk/en/) is a high-performance PPK/RTK positioning module created by CUAV for professional applications such as drone aerial survey (Mapping).
+It has a high-precision IMU and positioning module.
+It can reduce more than 80% of control points.
```

Comment:
What does this mean? What is a control point, and relative to what are you reducing them?

I am "guessing" this means something to do with a mission survey, and how many waypoints, but it isn't clear and it isn't common terminology.

Reply:
[control point](https://www.youtube.com/watch?v=z6SXLOZEyvk)；Control points are a means of correcting position errors in aerial surveying

Reply:
Thanks. So you control points essentially provide an accurate position mapping on the image. I think you're saying that you reduce the need for these because the images are geotagged to a high level of precision using this GPS module? Because of that the work for someone adding the points manually is reduced?

Probably best just to provide a link here to the definition. How about this?:
```suggestion
It can reduce the number of [control points](https://www.groundcontrolpoints.com/#yui_3_17_2_1_1645072055305_632) required to correct geotagging errors by more than 80%.
```